### DISCUSSION TITLE
Save live timing layout in setups

### DISCUSSION THREAD

**Discussion by Arnoud-B** [f1multiviewer/issue-tracker#442] [Feature requests] [answered]
Now that we can save the live timing screen location and size through saved setups, it would be great to also store the selected living timing layout in the setup.

I have separate living timing layouts for sessions and races, for instance, so I'd love to be able to open everything from the saved setup in one go and not having to switch layout manually after loading the saved setup.

**Comment by f1multiviewer** [comment, marked as answer]
It's actually saved, the restore function doesn't work correctly so I'll look into it

**Comment by Arnoud-B** [comment]
Fixed today, so closing this one. Thanks for the quick fix 🙏
